### PR TITLE
Add pagination to getOwnersForNft

### PR DESCRIPTION
**Pull request by joshzhang5**

### CODE REVIEW

**Comment by joshzhang5** [comment]
Current dependencies on/for this PR:
* main
  * **PR #293** <a href="https://app.graphite.dev/github/pr/alchemyplatform/alchemy-sdk-js/293" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/alchemyplatform/alchemy-sdk-js/293?utm_source=stack-comment).

File: docs-md/interfaces/GetOwnersForNftResponse.md [hunk 1]
```diff
@@ -18,6 +18,12 @@ The response object for the [getOwnersForNft](../classes/NftNamespace.md#getowne
 
 An array of owner addresses for the provided token.
 
+### pageKey (optional)
```
Comment (thebrianchen) [inline docs-md/interfaces/GetOwnersForNftResponse.md:21]: Can you revert this change? We usually only regenerate the SDK documentation when creating a new release

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "alchemy-sdk",
-  "version": "2.6.2",
+  "version": "2.6.3",
```
Comment (thebrianchen) [inline package.json:3]: don't bump the version just yet, i'll do that in a separate release. thanks!

**Review by thebrianchen — COMMENTED** [review]
looking good! thanks for adding all the tests! can you just revert the `.md` file changes and the version bump?

File: .gitignore [hunk 3]
```diff
@@ -16,3 +16,6 @@ temp
 alchemy.env
 
 .npmrc
+
+# VSCode History Extension
+.history
```
Comment (thebrianchen) [inline .gitignore:21]: nit: add a new line

File: src/types/types.ts [hunk 4]
```diff
@@ -1598,6 +1601,24 @@ export interface GetOwnersForContractWithTokenBalancesOptions {
   pageKey?: string;
 }
 
+/**
+ * Optional parameters object for the {@link getNftsForContract} method.
```
Comment (thebrianchen) [inline src/types/types.ts:1605]: s/getNftsForContract/getOwnersForNft

File: src/types/types.ts [hunk 5]
```diff
@@ -1598,6 +1601,24 @@ export interface GetOwnersForContractWithTokenBalancesOptions {
...
+ *
+ * This interface configures options when fetching the owner addresses of the
+ * provided contract.
```
Comment (thebrianchen) [inline src/types/types.ts:1608]: s/contract/NFT

**Review by thebrianchen — APPROVED** [review]
woot! 🚀 thanks for adding it to the SDK! left a couple of nits i just noticed
